api/catalog/api/controllers/search_controller.py

```diff
@@ -407,10 +406,9 @@ def search(
     try:
         if settings.VERBOSE_ES_RESPONSE:
             log.info(pprint.pprint(s.to_dict()))
+
         search_response = s.execute()
-        log.info(
-            f"query={json.dumps(s.to_dict())}," f" es_took_ms={search_response.took}"
-        )
```

Comment:
The `es_took_ms` one I have found useful on occasion; maybe we could just remove the search dumping? Not a blocker though, we can add it back in a different issue in a way that is less verbose.